tests/serverpod_test_server/test/generated_methods/to_json_test.dart

```diff
@@ -610,4 +610,23 @@ void main() {
       });
     });
   });
+
+  test(
```

Comment:
Yeah you are right, we do have some e2e / integration tests that cover both ends but these unit tests only cover the serverside implementation (even if they should be almost the same, except for the allToJson).

Worth duplicating the test suit to cover the client too? Maybe?

Reply:
Yeah, not sure on this one. It's quite a large test suite that would need to be maintained on both sides. Maybe it would be possible to run the suite twice here directly with the different implementations?

Otherwise it would be nice to just get alerted if we ever split the implementations apart.

Reply:
If we had a base object we could do that but since we do not It is not possible as we need separate imports unfortunately.

Reply:
😞